packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -507,11 +523,20 @@ class Auth {
             this.logWarning(SLAS_SECRET_WARNING_MSG)
         }
         const usid = this.get('usid')
+        const dw_dnt = this.get('dw_dnt')
+        const dntPref =
+            dw_dnt === '1'
+                ? true
+                : dw_dnt === '0'
+                ? false
+                : this.defaultDnt !== undefined
+                ? this.defaultDnt
+                : undefined
```

Comment:
This code chunk seems repetitive. Do you want to get them into a seperate func? e.g `getDntPref`